Plugins/GitFlow/GitFlowForm.cs

```diff
@@ -152,9 +152,9 @@ private void DisplayBranchDatas()
             }
 
             btnFinish.Enabled = isThereABranch &&(branchType != Branch.support.ToString("G"));
-            btnPublish.Enabled = isThereABranch;
-            btnPull.Enabled = isThereABranch;
-            pnlPull.Enabled = (branchType == Branch.feature.ToString("G"));
+            btnPublish.Enabled = isThereABranch && (branchType != Branch.support.ToString("G"));
+            btnPull.Enabled = isThereABranch && (branchType != Branch.support.ToString("G"));
+            pnlPull.Enabled = (branchType != Branch.support.ToString("G"));
```

Comment:
There is no real need to specify "G" (https://docs.microsoft.com/en-us/dotnet/standard/base-types/enumeration-format-strings), it is used by default if format not specified.
Not an issue, just an observation